README.md

```diff
@@ -57,3 +57,6 @@ Fastlane Flexible Integration allows you to customize and style your payment pag
 #### Key Features
 - Further customize the behavior and experience of your checkout
 - Data Security: Flexible Integration is PCI DSS compliant, ensuring that customer payment information is handled securely
+
+## Codespaces
```

Comment:
@louislowjk  can you update this readme with content similar to this 
https://github.com/paypal-examples/googlepay?tab=readme-ov-file#paypal-codespaces 

Please add all the content under the PayPal codespaces header and modify the codespaces link as peer this repo. 

This was approved by our content team so wanted to maintain the same for consistency. thanks

Reply:
Updated.